### PR TITLE
updated infosheet to display the lablel value on the infosheet when a…

### PR DESCRIPTION
**Pull request by Adamws33**
updated infosheet to display the label info if one exists.

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.228.100:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.228.100:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/shared/constants.js [hunk 1]
```diff
@@ -1080,7 +1080,12 @@ export const extractData = (attribute, from) => {
   let data;
   let dataType;
   if (from === 'infoSheet') {
-    data = attribute?.values[0]?.label ? attribute?.values[0]?.label : attribute?.values[0]?.value;
```
Comment (mikahpinegar) [inline src/shared/constants.js:1083]: this should already handle it properly right? data gets returned in a typography on line 1120

Comment (mikahpinegar) [reply src/shared/constants.js:1083]: this is still broken but i think it's now an api thing. @trevorjpuckett in the crops endpoint the label is showing up properly but in the info sheet endpoint crop.data[index].attributes.values[0] has label as null for everything

Comment (trevorjpuckett) [reply src/shared/constants.js:1083]: @mikahpinegar looks like some are showing up properly, while others are not. this looks like its an error in the dataset. I will have this fixed by EoD today!

**Review by mikahpinegar — APPROVED** [review]

**Pull request merged**